members/migrations/0001_initial.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2020-12-20 23:29
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import members.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("website", "0009_auto_20201220_1811"),
```

Comment:
where does this come from/what is it doing? can we add a comment?

Reply:
we could add a comment, but the documentation for migrations is pretty solid. It comes from `websites` migrations (`website/migrations/0009_auto_20201220_1811.py`).

Reply:
mmm i see- thanks!